### ISSUE TITLE
CRTs: NFT sale proceeds did not register in CRT revenue account

### ISSUE THREAD

**Issue by mochet** [Joystream/joystream#5180]
# Problem
* I sold this NFT in an auction for 17.5k $JOY: https://gleev.xyz/video/1411116
* I settled the auction in this TX: https://joystream.subscan.io/extrinsic/8697584-1
* I got a notification about royalties
![image](https://github.com/user-attachments/assets/fee07702-2ebd-4e94-9450-4ff3aac22e8f)
* My channel only shows a revenue of 875 $JOY
 ![image](https://github.com/user-attachments/assets/95279274-ed39-45e6-8208-9337b4f43d44)
* The actual amount in the revenue account is 17.15k $JOY: https://joystream.subscan.io/account/j4To6jgKfy8MgiMRDXzc15ggN8AnFBYTr7QyXyGc5B1T8MM7a

# Solution
I guess Orion needs to be adjusted to account for this type of revenue?

**Comment by bedeho** [comment]
What do you mean "this type of revenue", do you just mean NFT royalties? If so then I think it already was, @WRadoslaw can weigh in.

**Comment by mochet** [comment]
> What do you mean "this type of revenue", do you just mean NFT royalties? If so then I think it already was, @WRadoslaw can weigh in.

No, it registered the royalties but it didn't register the revenue from the actual NFT sale which seem to be 2 different things.
* Revenue = 875 $JOY (registered)
* Sale = 16,275 $JOY (didn't show up--from extrinsic https://joystream.subscan.io/block/8697584?tab=event&event=8697584-4)

**Comment by bedeho** [comment]
Oh ok, well that def. is supported, so not sure what happened. NFT sales was arguably the most important rev. source contemplated.
